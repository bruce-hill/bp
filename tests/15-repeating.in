(1234)
(abcd)
(abcd)
(;;;;)
(12)
(ab)
(x)
(1234567)
