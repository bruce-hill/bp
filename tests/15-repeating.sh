# Numbers allow you to specify repetitions of a pattern
# Example: bp -p '3 "x"' matches "xxx"
bp -p '`( 4 . `)'
