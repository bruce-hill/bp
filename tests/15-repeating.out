(1234)
(abcd)
(abcd)
(;;;;)
